Utilities/DataLoading/EdFi.LoadTools/Mapping/MapperConfigurationExpression.cs

```diff
@@ -115,19 +116,21 @@ private static IEnumerable<PropertyInfoEx> GetRecursiveProperties(Type type, str
                     continue;
                 }
 
+                var propertyPath = $"{path}{Constants.PropertyPathSeparator}{propertyInfo.Name}".Trim(Constants.PropertyPathSeparator);
```

Comment:
Trim() is needed because path can be an empty string and we don't want leading slashes.